pkg/proc/variables.go

```diff
@@ -1111,7 +1111,7 @@ func readVarEntry(entry *godwarf.Tree, image *Image) (name string, typ godwarf.T
 
 // Extracts the name and type of a variable from a dwarf entry
 // then executes the instructions given in the  DW_AT_location attribute to grab the variable's address
-func extractVarInfoFromEntry(bi *BinaryInfo, image *Image, regs op.DwarfRegisters, mem MemoryReadWriter, entry *godwarf.Tree) (*Variable, error) {
+func extractVarInfoFromEntry(tgt *Target, bi *BinaryInfo, image *Image, regs op.DwarfRegisters, mem MemoryReadWriter, entry *godwarf.Tree) (*Variable, error) {
```

Comment:
We can remove the `bi *BinaryInfo` argument here now that we're passing in `Target`.

Reply:
We can't, at the moment. When it gets called to evaluate a breakpoint condition tgt will be nil because that code doesn't have access to a target object. I was thinking of moving the breakpoint condition evaluation code into `(*Target).Continue` but it's too late in the release cycle and it overlaps too much with #2519. I think we have to live with this uglyness for a while.

Reply:
Ack, alright. I'm on board with shifting around the breakpoint condition eval code as well.